app/serializers/service_provider_serializer.rb

```diff
@@ -17,7 +17,7 @@ class ServiceProviderSerializer < ActiveModel::Serializer
   )
 
   def agency
-    object.name
+    object.friendly_name
```

Comment:
Oh, shoot. Should be `agency`, so yes, can be removed. And same with `def friendly_name`, can also be removed — `description` is only used in the dashboard for agencies to add context to their SP entry. Thanks!